client/src/components/MyAccount/AddItemFormSteps/StepThree.jsx

```diff
@@ -116,6 +137,110 @@ const StepThree = ({
           />
         </div>
 
+        <label className="text-lg leading-7">Payment details</label>
```

Comment:
Didn't we say we can remove this part when advertising product, we need it only when paying for it?